repository/k.json

```diff
@@ -23,6 +23,18 @@
 				}
 			]
 		},
+		{
+			"name": "Ebitengine Kage",
+			"details": "https://github.com/sedyh/ebitengine-kage-support/tree/main/sublime",
```

Comment:
Does that work? Normally you submit a repo, not a path inside a repo.

Reply:
This repository contains all versions of the plugin for different editors. Initially, I thought that I could specify where to get the files to build the archive from, but here it does not work.
I tested the package installation a bit. I could create a separate branch for sublime text and remove other subfolders or add .gitattributes. Are branch based releases still supported?

Reply:
Ok, I've split all my subfolder to separate repos and moved this one to e.json.